code/test/sixsq/nuvla/server/resources/infrastructure_service_group_lifecycle_test.clj

```diff
@@ -161,22 +161,23 @@
                   (request (str p/service-context service-id)
```

Comment:
There must be a verification that infrastructure service group resources can be deleted.  Either mock the work of the jobs or do not create jobs (and a 202 error response) when not necessary.

Reply:
back to the original code now and working